source/_patterns/04-pages/article--research.mustache

```diff
@@ -1,4 +1,4 @@
-<div class="global-wrapper">
+<div class="global-wrapper"  data-item-type="default">
```

Comment:
I know they result in the same output, but to match whats on prod it should be 'research-article'.

Reply:
I have changed the one for the article section.